src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -64,6 +64,10 @@ internal sealed class SmartRenameViewModel : INotifyPropertyChanged, IDisposable
         }
     }
 
+#pragma warning disable CA1822 // Mark members as static - used in xaml
+    public string GetSuggestionsTooltip => EditorFeaturesWpfResources.Get_AI_suggestions;
```

Comment:
I tried it but it doesn't work because Get_AI_suggestions was considered a field by the compiler, even though it's actually a property. I wonder if it's because it was mark as internal instead of public.

Reply:
Ah, yes, it has to be public to be used in XAML. Either way is fine for me.